content/news/2024-02-03-bevy-0.13/index.md

```diff
@@ -834,11 +834,34 @@ commands.spawn((
 ));
 ```
 
-## Exposure Settings
+## Camera Exposure
 
-<div class="release-feature-authors">authors: @TODO</div>
+<div class="release-feature-authors">authors: @superdump (Rob Swain), @JMS55, @cart</div>
 
-TODO.
+Cameras in the "real world" control how bright an image is by adjusting "exposure settings":
```

Comment:
```suggestion
Cameras in the "real world" control how bright an image is by adjusting several "exposure settings":
```

Reply:
Isn't "several" implied by the plural "settings" and the list that follows? Is this a grammar rule I'm not aware of?

Reply:
"Settings" is the plural, yes; however, I think the usage here was more in line with "separate and distinct". That exposure is gotten by adjusting different settings in conjunction? Not sure if it's necessary here tho.

Reply:
Part of the broader confusion here for those unfamiliar with photography is "why do we need multiple"? "Can't we just set a brightness level?"

This was intended to push this section in that direction :) I should have been more clear, sorry.

Reply:
Why the square quotes around “real world” and “exposure settings”? 😄

Reply:
I do tend to be overzealous with the double quotes. I like using them to group and introduce concepts. I'll defend "exposure settings" in that context. But "real world" is straight up ambiguous and confusing :laughing:

Reply:
> Part of the broader confusion here for those unfamiliar with photography is "why do we need multiple"? "Can't we just set a brightness level?"

This is reasonable / I see the value now. I'm in!